src/main/resources/templates/downloads-11.ftl

```diff
@@ -0,0 +1,357 @@
+        </td>
+    </tr>
+    <tr>
+        <td>Overlay</td>
+        <td>Server add-on for WildFly. Not recommended in production.</td>
+        <td>
+            <@download category="overlay" label="standalone" file="keycloak-overlay-${version.version}" tar=true />
+        </td>
+    </tr>
+    <tr>
+        <td>Demo</td>
+        <td>Demo distribution <b>*DEPRECATED*</b></td>
+        <td>
+            <@download category="demo" label="demo" file="keycloak-demo-${version.version}" tar=true />
+        </td>
+    </tr>
+    </tbody>
+</table>
+
+
+<h2>Examples</h2>
+
+<table class="table table-bordered table-striped">
+    <tbody>
+
+    <tr>
+        <td>Quickstarts distribution</td>
+        <td>
+            <a onclick="dl('examples', 'quickstarts');" href="https://github.com/keycloak/keycloak-quickstarts" target="_blank">
+                <i class="fa fa-github" aria-hidden="true"></i>
+                GitHub
+            </a>
+            <span class="space"></span>
+            <a onclick="dl('examples', 'quickstarts');" href="https://github.com/keycloak/keycloak-quickstarts/archive/latest.zip" target="_blank">
+                <i class="fa fa-download" aria-hidden="true"></i>
+                ZIP
+            </a>
+        </td>
+    </tr>
+    <tr>
+        <td>Old examples <b>*DEPRECATED*</b></td>
+        <td>
+            <@download category="examples" label="examples" file="keycloak-examples-${version.version}" tar=false />
+        </td>
+    </tr>
+    </tbody>
+</table>
+
+
+<h2>Client Adapters</h2>
+
+<div>
+    <ul class="nav nav-tabs" role="tablist">
+        <li role="presentation" class="active"><a href="#oidc" aria-controls="oidc" role="tab" data-toggle="tab">OpenID Connect</a></li>
+        <li role="presentation"><a href="#saml" aria-controls="saml" role="tab" data-toggle="tab">SAML 2.0</a></li>
+    </ul>
+
+    <div class="tab-content">
+
+        <div role="tabpanel" class="tab-pane active margin-top" id="oidc">
+            <table class="table table-bordered table-striped">
+                <tr>
+                    <td>Keycloak Proxy <b>*DEPRECATED*</b></td>
+                    <td>
+                        <table class="table-downloads-inner">
+                            <tr>
+                                <td>&nbsp;</td>
+                                <td>
+                                    <@download category="proxy" label="proxy" file="keycloak-proxy-${version.version}" tar=false />
+                                </td>
+                            </tr>
+                        </table>
+                    </td>
+                </tr>
+                <tr>
+                    <td>WildFly</td>
+                    <td>
+                        <table class="table-downloads-inner">
+                            <tr>
+                                <td>
+                                   13<br />
```

Comment:
Could the 11, 10, 9 be in separate cell (similarly to 8) instead of <br>?

Reply:
@hmlnarik Maybe? Do we want to emphasize it's same ZIP for all 9-13 or not emphasize and use separate cells? I don't have strong preference. @stianst WDYT?

Reply:
Do we want to somehow mention we doesn't support wf12 at all?

Reply:
I'm happy with just <br> and showing that it's the same DL.

I'd just leave WF12 out of it. Trying to mention that would mess things up.

Reply:
I don't have strong preference here too.

Reply:
I'm ok with it as it's now.